.travis.yml

```diff
@@ -18,7 +18,6 @@ addons:
     - george-edison55-precise-backports # cmake 3.2.3
 env:
   - LLVM_VERSION=3.9.0
-before_install: . vendor/bindgen/ci/before_install.sh
```

Comment:
I've since removed this from `master` :)

Reply:
Nice! Rebase it is